Add content to README (#1)
snugnug/.github: profile/README.md

# Hello, ~my~ *our* name is "SNUG"

> We are a small group of nix enthusiasts from many different parts of the world, trying to contribute back to one of our favourite ecosystems.

Reviewer: ```suggestion
We are a small group of Nix enthusiasts from many different parts of the world, trying to contribute back to one of our favorite ecosystems.
```

# Top Projects

> ## [Hjem-rum](https://github.com/snugnug/hjem-rum)
> A home-manager-like module system built on top of [hjem](https://github.com/feel-co/hjem)
> 
> ## [micros](https://github.com/snugnug/micros)
> A small operating system, aiming to bring nixos-like declaritivity to embedded systems

Reviewer: ```suggestion
## [Hjem Rum](https://github.com/snugnug/hjem-rum)
A Home Manager-like module system built on top of [Hjem](https://github.com/feel-co/hjem).

## [MicrOS](https://github.com/snugnug/micros)
A small operating system, aiming to bring NixOS-like declarative to embedded systems.
```

Author: Why'd you replace 'declarativity' with 'declarative' in line 10? Im pretty sure that 'declarative' is a verb, making it unfit to appear in that position

Reviewer: 'twas a mistake, but yours was misspelled. should be "declarativity"

Reviewer: ```suggestion
## [Hjem Rum](https://github.com/snugnug/hjem-rum)
A Home Manager-like module system built on top of [Hjem](https://github.com/feel-co/hjem).

## [MicrOS](https://github.com/snugnug/micros)
A small operating system, aiming to bring NixOS-like declarativity to embedded systems.
```

Author: Ah, I didn't even notice. Thanks!

## [sherpa](https://github.com/snugnug/sherpa)

> Work-In-Progress effort to provide a beginner-friendly guide to the world of nix and nixos

Reviewer: ```suggestion
WIP effort to provide a beginner-friendly guide to the world of Nix and NixOS.
```

> [!Note]
> This could really use some more content.

# Credit

> Please see [the dedicated file](../assets/README.md)

Reviewer: ```suggestion
Please see [the dedicated file](../assets/README.md).
```
